code/game/turfs/space/space.dm

```diff
@@ -54,12 +54,12 @@
 				qdel(A)
 				return
 
-			var/specials_list = recursive_type_check(A, list(/obj/item/weapon/disk/nuclear, /mob/living/silicon/robot/mommi))
+			var/specials_list = search_contents_for(A, list(/obj/item/weapon/disk/nuclear, /mob/living/silicon/robot/mommi))

```

Comment:
Looking at the code, that isn't what it does.
Arg 1: Search for this general type
Arg 2: Anything not of a type in this list will not have its contents searched.